lasagne/layers/recurrent.py

```diff
@@ -944,6 +944,7 @@ def get_output_for(self, inputs, **kwargs):
         """
         # Retrieve the layer input
         input = inputs[0]
+        print input
```

Comment:
Uhm, yeah, my bad, didn't realize I was still on that branch while snooping around.